src/library/scala/collection/Iterable.scala

```diff
@@ -133,6 +135,13 @@ trait IterableOps[+A, +CC[_], +C] extends Any with IterableOnce[A] with Iterable
     */
   def toIterable: Iterable[A]
 
+  /** Converts this $coll to an unspecified Iterable.  Will return
+    *  the same collection if this instance is already Iterable.
+    *  @return An Iterable containing all elements of this $coll.
+    */
+  @deprecated("Use toIterable instead", "2.13.0")
+  def toTraversable: Traversable[A]
```

Comment:
Why is this abstract? Could we define `@deprecated(..) final def toTraversable: Traversable[A] = toIterable` here?

Reply:
That makes sense.  I was just mimicking the definition of `toIterator`.  I gave it a try and removed the abstract definition.  It actually makes this change a lot more minimal.

I don't think there's any reason it needs to be abstract.  Actually, it looks like overriding `toTraversable` was marked deprecated in 3cc99d7.  It wasn't possible to actually make it final in 2.12, because it was still being overridden in `ParIterableLike` and `TraversableProxyLike`, but we can try to do that here now.

Reply:
I second @lrytz, just define it once and implement it at the same time.